ts/utils/hooks/bottomSheet.tsx

```diff
@@ -56,78 +41,27 @@ const styles = StyleSheet.create({
     overflow: "hidden"
   }
 });
-/**
- * Build the base content of a BottomSheet including content padding and a ScrollView
- */
-const BottomSheetContent: FunctionComponent<Props> = ({
-  children,
-  testID
-}: Props) => (
-  <BottomSheetScrollView
-    style={{
-      flex: 1,
-      paddingHorizontal: IOVisualCostants.appMarginDefault
-    }}
-    testID={testID}
-  >
-    {children}
-  </BottomSheetScrollView>
-);
-
-export type BottomSheetModalProps = {
-  content: ReactNode;
-  config: {
-    handleComponent: ReactElement;
-  };
-};
 
 export type IOBottomSheetModal = {
   present: () => void;
   dismiss: () => void;
   bottomSheet: JSX.Element;
 };
 
-/**
- * Utility function to build a BottomSheet considering accessibility. This will create a common BottomSheet object to be used in the `present` function
- * that is available only in component context since it uses the context api made available from https://gorhom.github.io/react-native-bottom-sheet/modal/methods
- * @param content
- * @param title
- * @param snapPoint
- * @param onClose
- */
-export const bottomSheetContent = (
-  content: ReactNode,
-  title: string | ReactNode,
-  onClose: () => void
-): BottomSheetModalProps => {
-  const header = <BottomSheetHeader title={title} onClose={onClose} />;
-
-  const bottomSheetBody: ReactNode = (
-    <BottomSheetContent>{content}</BottomSheetContent>
-  );
-
-  return {
-    content: bottomSheetBody,
-    config: {
-      handleComponent: header
-    }
-  };
-};
-
 /**
  * @typedef BottomSheetOptions
  * @type {BottomSheetOptions}
- * @property {component} component The Element to be rendered inside the bottom sheet body
- * @property {title} title String or Element to be rendered as bottom-sheet header title
- * @property {footer} footer Element or undefined to be rendered as sticky footer of our bottom sheet
+ * @property {component} component The React.Element to be rendered inside the bottom sheet body
+ * @property {title} title String or React.Element to be rendered as bottom-sheet header title
+ * @property {footer} footer React.Element or undefined to be rendered as sticky footer of our bottom sheet
  * @property {snapPoint} snapPoint The array of points used to pin the height of the bottom sheet
  * @property {onDismiss} onDismiss The possible function to be used as an effect of the dismissal of the bottom sheet
  */
 type BottomSheetOptions = {
-  component: ReactNode;
-  title: string | ReactNode;
-  snapPoint: NonEmptyArray<number | string>;
-  footer?: ReactElement;
+  component: React.ReactNode;
+  title: string | React.ReactNode;
+  snapPoint?: NonEmptyArray<number | string>;
+  footer?: React.ReactElement;
```

Comment:
Consider simplifying the type definitions by importing the necessary types directly from react

```suggestion
  component:ReactNode;
  title: string |ReactNode;
  snapPoint?: NonEmptyArray<number | string>;
  footer?: ReactElement;
```